pmoves/services/messaging-gateway/platforms/whatsapp.py

```
@@ -0,0 +1,120 @@
+    async def initialize(self):
+        """Initialize HTTP client."""
+        if not self.is_configured():
+            return
+
+        self._client = httpx.AsyncClient(timeout=15)
+        logger.info("WhatsApp platform initialized (stub)")
+
+    async def send(
+        self,
+        content: str,
+        buttons: Optional[List[Dict]] = None,
```

Ruff (0.14.8)

Unused method argument: `buttons`

(ARG002)